prov/sm2/src/sm2_rma.c

```diff
@@ -0,0 +1,499 @@
+/*
+ * Copyright (c) 2013-2021 Intel Corporation. All rights reserved
```

Comment:
Is it because we are copying it from shm?

Reply:
Yes.  Originally this was smr_rma.c.  Much of it has changed, but not all of it.